### ISSUE TITLE
Poll Forms

### ISSUE THREAD

**Issue by visionalx** [yegnaro/OurVotes#1]
How many Questions and options per form?

**Comment by yegnaro** [comment]
I was originally thinking of a tab for each race. Reference the actual Voter Ballot for specifics.

Tabs:
1. President
2. Senate Seats
3. House of Representatives Seats
4. Local Race Seats
5. Propositions

**Comment by visionalx** [comment]
Yes, lets also take inconsideration unconventional polls.  Sometimes organizations choose their boards by popularity votes; the member with the highest votes becomes President, second highest VP, third Treasurer, fourth Secretary.... and so on.

**Comment by yegnaro** [comment]
I think I am on the same page as you. With that said, we should have multiple types of polls then as an option, available on the admin pages, when creating the polls.

**Comment by visionalx** [comment]
Nice, another one is selecting some sort of community committee.  Out of 20 names only 10 with the highest votes become members...   its a good start.

**Comment by yegnaro** [comment]
I think the two mentioned by @visionalx can be the same one. User variables to show X results for admins.
